include/dts-functions.sh

```diff
@@ -1230,6 +1230,8 @@ You can find more info about HCL in docs.dasharo.com/glossary\r"
 }
 
 show_ram_inf() {
+  # trace logging is quite slow due to timestamp (calls 'date')
```

Comment:
It causes those delays because it's external program called for each line in subshell.
Each refresh generates about 800 logged lines of which about 660 is a result of this function (mostly because of parsing long output in loop).

Reply:
Could you make a comment in the function using your response?

Reply:
965855f749476003b9040eb7a73a2a14725d52eb

We don't need to disable trace logging but temporarily change PS4 variable value so it doesn't contain call to date but I'm not sure if we want 650 lines of spam for every screen refresh (which increases log size by 60 KB for each refresh instead of just around 7 KB)